src/docs/index.rst

```diff
@@ -182,6 +194,35 @@ We provide the following useful build targets that can be run from the build dir
 * ``style``: Runs styling over source code and replaces files in place
 * ``check``: Runs a set of code checks over source code (CppCheck and clang-format)
 
+Preparing Windows WSL/Ubuntu for Serac installation
+---------------------------------------------------------
+
+For faster installation of the Serac dependencies via Spack on Windows WSL/Ubuntu systems, install cmake, MPICH, openblas, OpenGL, and the devtools
+using the following commands for Ubuntu 20.04:
+
+.. code-block:: bash
+
+   $ sudo apt-get update
+   $ sudo apt-get upgrade
+   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev python3-sphinx clang-format-10
+   $ sudo ln -s /usr/lib/x86_64-linux-gnu/* /usr/lib
+
+and the following commands for Ubuntu 18.04:
+
+.. code-block:: bash
+
+   $ sudo apt-get update
+   $ sudo apt-get upgrade
+   $ sudo apt-get install g++-8 gcc-8
+   $ sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+   $ sudo apt-get install cmake libopenblas-dev libopenblas-base mpich mesa-common-dev libglu1-mesa-dev freeglut3-dev cppcheck doxygen libreadline-dev
+   $ sudo ln -s /usr/lib/x86_64-linux-gnu/* /usr/lib
```

Comment:
This should probably go back to the spack group so they can handle this behind the scenes for people.  @becker33